modules/exploits/linux/http/tr064_ntpserver_cmdinject.rb

```diff
@@ -104,6 +104,7 @@ def execute_command(cmd, opts)
   def check
     begin
       res = send_request_cgi({
+'method' => 'GET',
```

Comment:
On running ./tools/dev/msftidy.rb modules/
This warning pops out:-
[WARNING] Tabbed indent: "\t'method' => 'GET'\n"